docs/serving/app-security/security-guard-install.md

```diff
@@ -1,20 +1,25 @@
 # Installing Security-Guard
 
+!!! note
+    Knative operator supports installing Security-Guard. This guide is for users installing Security-Guard without the Knative operator.
```

Comment:
Has this merged yet? I couldn't find the PR for it, so if it's still WIP let's remove this note and add back in once it merges.